sources/plugins/LametricTime/locales/en.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "LametricTime",
+  "description": "Lametric devices support",
```

Comment:
Prefer "Lametric Time devices support", and add a link to Lametric Time official page (https://lametric.com/en-US/time/overview)